### PR TITLE
qgis3: fix qspatialite

### PR DESCRIPTION
**Pull request by fjperini**
Solution for the problem reported in #434.

There was not a reference of `${QT_PLUGINS_DIR}/sqldrivers` in `src/providers/spatialite/CMakeLists.txt
`

Checking I found

```
qsqlspatialite
external/qspatialite/CMakeLists.txt
${QT_PLUGINS_DIR}/sqldrivers

spatialiteprovider
src/providers/spatialite/CMakeLists.txt
${QGIS_PLUGIN_DIR}
```

### CODE REVIEW

**Comment by nickrobison** [comment]
@fjperini Have you verified this builds correctly on your local machine?

**Comment by fjperini** [comment]
@nickrobison Yes!

File: Formula/qgis3.rb [hunk 1]
```diff
@@ -312,8 +314,10 @@ def install
     # install db plugins to local qt plugins prefix
     if build.with? "qspatialite"
       mkdir lib_qt/"plugins/sqldrivers"
-      inreplace "src/providers/spatialite/CMakeLists.txt",
-                "${QT_PLUGINS_DIR}/sqldrivers", lib_qt/"plugins/sqldrivers".to_s
+      # inreplace "src/providers/spatialite/CMakeLists.txt",
```
Comment (nickrobison) [inline Formula/qgis3.rb]: In the future, please don't comment out old code, it makes the formula harder to read, and we can get the diffs via the git log, if we need to.

Comment (fjperini) [reply Formula/qgis3.rb]: You are right! sometimes I leave them for me :joy:

**Pull request merged**